fabric-convention-tags-v2/src/datagen/java/net/fabricmc/fabric/impl/tag/convention/datagen/generators/BlockTagGenerator.java

```diff
@@ -226,12 +228,12 @@ private void generateFlowerTags() {
 						Blocks.CLOSED_EYEBLOSSOM
 				);
 
-		getOrCreateTagBuilder(ConventionalBlockTags.TALL_FLOWERS)
+		valueLookupBuilder(ConventionalBlockTags.TALL_FLOWERS)
 				.add(Blocks.SUNFLOWER, Blocks.LILAC, Blocks.PEONY,
 						Blocks.ROSE_BUSH, Blocks.PITCHER_PLANT
-				).addOptional(Identifier.ofVanilla("tall_flowers"));
+				).addOptionalTag(TagKey.of(RegistryKeys.BLOCK, Identifier.ofVanilla("tall_flowers")));
```

Comment:
This is wrong, previously it was andding an option entry for this, and not an optional tag. I don't see any reference to `tall_flowers` in the game, so might just remove this?